lib/wired/builders/app_builder.rb

```diff
@@ -160,8 +174,8 @@ def test_configuration_files
 
     def setup_git
       run 'git init'
-      run "git add ."
-      run "git commit -m 'initial commit'"
+      run "git add -A"
+      run "git commit -am 'initial commit'"
       run "git checkout -b develop"
     end
 
```

Comment:
Misschien verstandig om vroeg in het proces te checken of alle benodigde tools aanwezig zijn, in plaats van te halverwege ermee op te houden. (In ieder geval zijn `hub` en `powder` vereist).